articles/spring-cloud/how-to-configure-enterprise-spring-cloud-gateway.md

```diff
@@ -0,0 +1,168 @@
+titleSuffix: Azure Spring Apps Enterprise Tier
+description: How to Configure Spring Cloud Gateway for Kubernetes with Azure Spring Apps Enterprise Tier.
+author: karlerickson
+ms.author: xiading
+ms.service: spring-cloud
+ms.topic: how-to
+ms.date: 02/09/2022
+ms.custom: devx-track-java, devx-track-azurecli, event-tier1-build-2022
+---
+
+# Configure Spring Cloud Gateway for Kubernetes
+
+> [!NOTE]
+> Azure Spring Apps is the new name for the Azure Spring Cloud service. Although the service has a new name, you'll see the old name in some places for a while as we work to update assets such as screenshots, videos, and diagrams.
+
+**This article applies to:** ❌ Basic/Standard tier ✔️ Enterprise tier
+
+This article shows you how to configure VMware Spring Cloud Gateway for Kubernetes with Azure Spring Apps Enterprise Tier.
+
+[Spring Cloud Gateway for Kubernetes](https://docs.vmware.com/en/VMware-Spring-Cloud-Gateway-for-Kubernetes/index.html) is one of the commercial VMware Tanzu components. It's based on the open-source Spring Cloud Gateway project. Spring Cloud Gateway for Tanzu handles cross-cutting concerns for API development teams, such as single sign-on (SSO), access control, rate-limiting, resiliency, security, and more. You can accelerate API delivery using modern cloud native patterns, and any programming language you choose for API development.
+
+Spring Cloud Gateway for Tanzu also has the following features:
+
+- Other commercial API route filters for transporting authorized JSON Web Token (JWT) claims to application services.
+- Client certificate authorization.
+- Rate-limiting approaches.
+- Circuit breaker configuration.
+- Support for accessing application services via HTTP Basic Authentication credentials.
+
+To integrate with [API portal for VMware Tanzu®](./how-to-use-enterprise-api-portal.md), Spring Cloud Gateway for Tanzu automatically generates OpenAPI version 3 documentation after the route configuration gets changed.
+
+## Prerequisites
+
+- An already provisioned Azure Spring Apps Enterprise tier service instance with Spring Cloud Gateway for Kubernetes enabled. For more information, see [Quickstart: Build and deploy apps to Azure Spring Apps Enterprise tier](quickstart-deploy-apps-enterprise.md#provision-a-service-instance).
+
+  > [!NOTE]
+  > To use Spring Cloud Gateway for Kubernetes, you must enable it when you provision your Azure Spring Apps service instance. You cannot enable it after provisioning at this time.
+
+- [Azure CLI version 2.0.67 or later](/cli/azure/install-azure-cli).
+
+## How Spring Cloud Gateway for Kubernetes works
+
+Spring Cloud Gateway for Kubernetes has two components: a Kubernetes operator and the Spring Cloud Gateway instance. The operator is responsible for the lifecycle of Spring Cloud Gateway instances and routing rules. It's transparent to the developer and Azure Spring Apps will manage it.
```

Comment:
This is too much implementation details. They are managed and developers need not worry